app/views/profiles/_education_fields.html.slim

```diff
@@ -0,0 +1,7 @@
+.nested-fields
+  .form-inline
+    = f.input :degree, label: false, placeholder: 'Your degreee'
+    = f.input :institution, label: false, placeholder: 'Your institution'
+    = f.input :graduating_year, as: :select, collection: 2013..2023, label: false
+    = link_to_remove_association f, data: {confirm:'do you really want to do this?'}, class: 'btn btn-primary btn-xs' do
```

Comment:
Comfirm dialog shoudl be more descriptive, smth like `Do you realy want to remove this education?`